scripts/Tools/pelayout

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+"""
+This utility allows the CIME user to view and modify a case's PE layout.
+
+Interpreted format sequences are:"
+%%  a literal %
+%C  the component name
+%T  the task count for the component
+%H  the thread count for the component
+%R  the PE root for the component
+
+Standard format extensions, such as a field length and padding are supported.
+Example:
+  For NTASKS_ATM=480, NTHREADS_ATM=2 and ROOTPE_ATM, \"%C: %06T/%+H\" will show
+ATM:    480/+2
```

Comment:
When I ran this example, I got a bunch of leading zeroes, in contrast to what you gave as example output:

```
CPL: 000015/+1
ATM: 000015/+1
LND: 000015/+1
ICE: 000015/+1
OCN: 000015/+1
ROF: 000015/+1
GLC: 000015/+1
WAV: 000015/+1
ESP: 000015/+1
```